src/Microsoft.AspNetCore.WebUtilities/BufferedReadStream.cs

```diff
@@ -39,36 +54,45 @@ public BufferedReadStream(Stream inner, int bufferSize, ArrayPool<byte> bytePool
             _buffer = bytePool.Rent(bufferSize);
         }
 
+        /// <summary>
+        /// The currently buffered data
```

Comment:
See earlier comment about ending sentences with periods. Please apply to all new doc comments.